mspa_sections.go

```diff
@@ -312,3 +372,440 @@ func (m *MspaUsCT) ParseConsent() (GppParsedConsent, error) {
 
 	return p, r.Err
 }
+
+func (m *MspaUsFL) ParseConsent() (GppParsedConsent, error) {
+	var segments = strings.Split(m.sectionValue, ".")
+
+	var b, err = base64.RawURLEncoding.DecodeString(segments[0])
+	if err != nil {
+		return nil, errors.Wrap(err, "parse usfl consent string")
+	}
+
+	var r = NewConsentReader(b)
+
+	// This block of code directly describes the format of the payload.
+	// The spec for the consent string can be found here:
+	// https://github.com/InteractiveAdvertisingBureau/Global-Privacy-Platform/tree/main/Sections/US-States/FL
+	var p = &MspaParsedConsent{}
+	p.Version, _ = r.ReadInt(6)
+
+	if p.Version != 1 {
+		return nil, errors.New("non-v1 string passed.")
+	}
+
+	// ProcessingNotice and SharingNotice are the same
```

Comment:
Please add comments in mspa_parsed_consent.go for this and AdditionalDataProcessingConents/PersonalDataConsents to reflect this information at the struct level.

Reply:
Done!